dts/bindings/sensor/raspberrrypi,pico-temp.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: RaspberryPi Pico family TEMP node
```

Comment:
Maybe expand TEMP to "temperature sensor"

Reply:
Corrected as your pointing out.